sdk/attestation/attestation/src/attestationAdministrationClient.ts

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
 /*
```

Comment:
Let's remove old comments

Reply:
I can't find this in my current code. Are you looking at an older version? I know I fixed this yesterday afternoon.

Reply:
Ah I see. I am going through the commits one by one. Ignore it then :)